lib/generator.js

```diff
@@ -63,8 +65,9 @@ class Generator {
    * @param {String[]} [options.noOverwriteGlobs] List of globs to skip when regenerating the template.
    * @param {String[]} [options.disabledHooks] List of hooks to disable.
    * @param {String} [options.output='fs'] Type of output. Can be either 'fs' (default) or 'string'. Only available when entrypoint is set.
+   * @param {String} [options.forceWrite] Force writing of the generated files to given directory even if it is a git repo with unstaged files or not empty dir.
    */
-  constructor(templateName, targetDir, { templatesDir, templateParams, entrypoint, noOverwriteGlobs, disabledHooks, output = 'fs' } = {}) {
+  constructor(templateName, targetDir, { templatesDir, templateParams, entrypoint, noOverwriteGlobs, disabledHooks, output = 'fs', forceWrite } = {}) {
```

Comment:
If the comment above makes sense, it could be:

```suggestion
  constructor(templateName, targetDir, { templatesDir, templateParams, entrypoint, noOverwriteGlobs, disabledHooks, output = 'fs', forceWrite = false } = {}) {
```

Reply:
look on my previous comment, I had it but removed on purpose. What you think?